documentation/create_overview_doc/README.md

```diff
@@ -51,21 +51,27 @@ sh
 kt
 ```
 
+For flow map generation with Code2Flow, the tool currently supports the following languages, each with specific dependencies:
+
+- JavaScript: Requires Acorn for parsing. Ensure Acorn is installed and accessible in your environment.
+- Ruby: Uses Parser for parsing. Ensure the Parser gem is installed and accessible.
+- PHP: Employs PHP-Parser for parsing. PHP-Parser should be installed and accessible in your environment.
+- Python: No additional dependencies are required for Python projects.
+
+If your project uses a language not supported by Code2Flow for flow map generation, the documentation generation capabilities will still be available, but without the flow map visualization for those specific languages.
```

Comment:
**Performance Issue**: Adding language-specific dependencies for Code2Flow could introduce potential bottlenecks, especially in environments where these dependencies are not optimized for performance. <br> **Fix**: Ensure that all language-specific dependencies are thoroughly tested for performance. Where possible, use more efficient parsing libraries or consider lazy loading of dependencies to reduce the initial load time. <br> **Code Suggestion**: 
 ```
 +For flow map generation with Code2Flow, the tool currently supports the following languages, each with specific dependencies: Ensure all language-specific dependencies are optimized for performance. Consider using more efficient parsing libraries or lazy loading.
 ```